src/elm/Page/Dashboard.elm

```diff
@@ -287,24 +287,7 @@ viewSections loggedIn model =
                 Page.viewCardEmpty [ text (t "community.actions.no_actions_yet") ]
 
             LoadedGraphql transfers ->
```

Comment:
@lucca65  perhaps the naming here should be changed to actions ...

Reply:
Hey @zacck this line wasn't changed, I've just let it as actions. To make this `actions` we actually have to create a new query, we'll have to discuss this in more depth first. I would say that this is out of the scope for this PR